uelc/main/models.py

```diff
@@ -66,11 +66,11 @@ def display_name(self):
         return '%s - %s' % (self.name)
 
     @classmethod
-    def create(self, cohort, hierarchy):
-        case = self(name=self.name,
-                    hierarchy=self.hierarchy,
-                    cohort=self.cohort)
-        return case
+    def create(self, request):
+        return Case.objects.create(
+            name=request.POST.get('name', ''),
+            hierarchy=request.POST.get('hierarchy', ''),
+            cohort=request.POST.get('cohort', ''))
 
```

Comment:
Yes, a cohort id will be passed in the args. going to remove this code for now because I have stuff on another branch that will take care of this.